packages/lib-react-components/src/Tooltip/components/Label/Label.js

```diff
@@ -1,17 +1,21 @@
-import PropTypes from 'prop-types'
+import { array, bool, node, object, oneOf, oneOfType, shape, string } from 'prop-types'
 import { Box } from 'grommet'
 import { withTheme } from 'styled-components'
+
 import SpacedText from '../../../SpacedText'
 import Triangle from '../Triangle'
 
-function Label ({ animation, arrow, className, label, theme, ...rest }) {
-  const { family } = theme.global.font
-  const placement = rest['data-placement'] // the attr from the render function of the tippy tooltip
+function Label ({ animation = 'fadeIn', arrow = true, className = '', label, theme, ...rest }) {
+
+  // Optional Chaining with a default param is necessary here to handle shallow rendered enzyme specs in lib-classifier
+  // In an app env, there will always be a Grommet theme wrapper
+  const family = theme?.global?.font?.family || ''
```

Comment:
This component still uses `withTheme` with `theme` as a prop because it's a child of several lib-classifier components with spec files using shallow Enzyme testing 😞 Those will need to be refactored to RTL or stories before Label can utilize `useTheme()` instead.